src/Datadog.Trace.AspNet/Datadog.Trace.AspNet.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45</TargetFrameworks>
+
+    <!-- NuGet -->
+    <Version>1.0</Version>
+    <Title>Datadog APM Tracing for ASP.NET</Title>
+    <Description>Provides tracing for ASP.NET web applications.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Datadog.Trace\Datadog.Trace.csproj" />
```

Comment:
I think one problem we had before when the HttpModule was in the same assembly as the rest of the integrations was that the initialization of the HttpModule caused the `Datadog.Trace.ClrProfiler.Managed` assembly to be loaded inside the AppDomain of that HttpApplication instead of being loaded domain-neutral. After this change, we'll need to test in IIS that the `Datadog.Trace` module is loaded domain-neutral (after installing into the GAC) and not loaded from the bin folder. I think we should be fine, but I wanted to bring this up because it's important that we get that right. Otherwise, `Datadog.Trace.ClrProfiler.Managed` (which depends on `Datadog.Trace`) cannot be loaded domain-neutral.

Reply:
We could use a pub/sub architecture `DiagnosticSource` and remove this dependency. Another small step in that direction.

Reply:
Actually, if we go that way, we just need to subscribe to event and we don't need this `HttpModule` at all to publish them because we can use [this package].

Reply:
Actually, if we go that way, we just need to subscribe to event and we don't need this `HttpModule` at all to publish them because we can use [this package].

Reply:
My initial comment here is the only concern I have left, just testing against an actual IIS environment to verify that this all comes together when adding the new package and that `Datadog.Trace.ClrProfiler.Managed` gets loaded domain-neutral.